app/controllers/pages_controller.rb

```diff
@@ -1,4 +1,11 @@
 class PagesController < ApplicationController
   def home
+    if current_user and current_user.recipes.blank? 
```

Comment:
This is a great addition! Now the user won't be directed to the log out screen if they had a previous session